spec/javascripts/components/story/expanded_story/expanded_story_state_spec.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ExpandedStoryState from 'components/story/ExpandedStory/ExpandedStoryState';
+import { states } from '../../../../../app/assets/javascripts/models/beta/story';
+
+describe('<ExpandedStoryState />', () => {
+  it('renders component title', () => {
+    const story = { state: 'started', _editing: { state: 'started' } };
+
+    const wrapper = shallow(<ExpandedStoryState story={story} />);
+
+    expect(wrapper.text()).toContain(I18n.t('activerecord.attributes.story.state'));
+  });
+
+  describe("states at select", () => {
+    states.map(state => {
```

Comment:
Is it really necessary to verify if all the available states are being selected? As long as they continue to be strings there will be no difference between them. What do you think about running a single test in this case?

Reply:
I agree with you, but, since I was undecided which one to test, I test all the cases just for guarantee  :information_desk_person: aahhaha.